code/_globalvars/traits.dm

```diff
@@ -156,6 +155,7 @@ GLOBAL_LIST_INIT(traits_by_type, list(
 		"Night Owl" = TRAIT_NIGHT_OWL,
 		"Ugly" = TRAIT_UGLY,
 		"Fishface" = TRAIT_FISHFACE,
+		"Obese" = TRAIT_FAT,
```

Comment:
make it so they are literally unable to climb and if the fall they just die hehe